### ISSUE TITLE
Deep Ocean Theme broken

### ISSUE THREAD

**Issue by DenuxPlays** [thvardhan/Gradianto#55]
Since the newest update (v5.5)
![image](https://github.com/thvardhan/Gradianto/assets/83671398/a716c9cc-855e-4e20-a7db-bbac8ad81fc6)

Editor and Terminal are white!!!!

**Comment by DenuxPlays** [comment]
This was already the case when applying the "new UI" theme to the new ui...
But I didn't care because you could still use the old version of it and it worked fine

**Comment by serembon** [comment]
@DenuxPlays Today I encountered the same problem. Solution provided in https://github.com/thvardhan/Gradianto/issues/50

TL;DR check `Color Scheme` option

<img width="674" alt="Screenshot 2023-10-26 at 16 35 28" src="https://github.com/thvardhan/Gradianto/assets/22729616/61f93453-481b-4e10-87d0-98cd6ade8d28">

**Comment by thvardhan** [comment]
<img width="642" alt="image" src="https://github.com/thvardhan/Gradianto/assets/17314986/d3bc007e-2d14-467a-b686-4ccb07a476d8">
Yes seems duplicate of #50 , its working fine on my end

**Comment by DenuxPlays** [comment]
> @DenuxPlays Today I encountered the same problem. Solution provided in #50
> 
> TL;DR check `Color Scheme` option
> 
> <img alt="Screenshot 2023-10-26 at 16 35 28" width="674" src="https://user-images.githubusercontent.com/22729616/278359150-61f93453-481b-4e10-87d0-98cd6ade8d28.png">

Worked! Thank you
